MyFirstPlugin/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.sonarsource</groupId>
+    <artifactId>MyFirstPlugin-parent</artifactId>
+    <version>0.1.0</version>
```

Comment:
You can use a `-SNAPSHOT` version here, Tycho will automatically change it to `.qualifier`